app-benchmarks/stress-ng/files/makefile.patch

```diff
@@ -0,0 +1,81 @@
+--- a/Makefile	2018-04-27 17:02:16.000000000 +0200
++++ b/Makefile	2018-05-12 23:32:30.075142809 +0200
+@@ -21,7 +21,7 @@
+ # Codename "portable pressure producer"
+ #
+ 
+-CFLAGS += -Wall -Wextra -DVERSION='"$(VERSION)"' -O2 -std=gnu99
++CFLAGS += -Wall -Wextra -DVERSION='"$(VERSION)"'
+ 
+ #
+ # Pedantic flags
+@@ -314,12 +314,9 @@
+ .o: stress-ng.h Makefile
+ 
+ .c.o: stress-ng.h Makefile $(SRC)
+-	@echo "CC $<"
+-	@$(CC) $(CFLAGS) -c -o $@ $<
+ 
+ stress-ng: info $(OBJS)
+-	@echo "LD $@"
+-	@$(CC) $(CPPFLAGS) $(CFLAGS) $(OBJS) -lm $(LDFLAGS) -lc -o $@
++	$(CC) $(CPPFLAGS) $(CFLAGS) $(OBJS) -lm $(LDFLAGS) -lc -o $@
+ 	@sync
+ 
+ .PHONY: info
+@@ -339,8 +336,7 @@
+ 		sed '$$ s/.$$//' >> apparmor-data.c
+ 	@echo "};" >> apparmor-data.c
+ 	@echo "const size_t g_apparmor_data_len = sizeof(g_apparmor_data);" >> apparmor-data.c
+-	@echo "CC $<"
+-	@$(CC) -c apparmor-data.c -o apparmor-data.o
++	$(CC) -c apparmor-data.c -o apparmor-data.o
```

Comment:
Missing CPPFLAGS, CFLAGS.

Reply:
Whoops, fixed this and rest. Could you tell me, why CPPFLAGS? Should I always add them?